ansible/files/bootstrap/telemetry.sh

```diff
@@ -73,11 +45,103 @@ while [ $# -gt 0 ]; do
     shift
 done
 
+
+CURRENT_KEY=`cat $HOME/.ssh/authorized_keys`
+TELEMETRY_CONF_BUCKET=s3://{{telemetry_analysis_spark_emr_bucket}}
+MEMORY_OVERHEAD=7000  # Tuned for c3.4xlarge
```

Comment:
We should  consider setting these values based on percentage of TotalMemory so that we can make use of other instance sizes. Although this out of scope for this specific PR and I am not going to block on it.